protobufs/account/get_user_data_rpc.proto

```diff
@@ -24,4 +25,5 @@ message GetUserDataResponse {
   string email_address = 12;
   string tier_name = 13;
   PNIDPermissionFlags permissions = 14;
+  repeated Device linked_devices = 15;
```

Comment:
This is the old v1 version of this protocol. We've moved on to v2 now, so v1 should be considered stable and not updated outside of bug fixes. If there are older clients that need new data, they should be upgrading imo, we shouldn't modify older versions of the protocol

Reply:
As far as I can tell, today's account server implements the v1 prototcol only? https://github.com/PretendoNetwork/account/blob/dev/src/services/grpc/account/get-user-data.ts

See:
```ts
import type { GetUserDataRequest, GetUserDataResponse } from '@pretendonetwork/grpc/account/get_user_data_rpc';
```

Wouldn't this be `@pretendonetwork/grpc/account/v2/get_user_data_rpc` if v2 was in use? If it's true that v2 is not implemented yet, I really would not like to wait for it since we this is an active security concern for Juxt, and this change is done in a backwards-compatible way.
If v2 is in use and I misunderstood something, no probs.